index.js

```diff
@@ -8,6 +8,7 @@ module.exports = pathtoRegexp;
  * Match matching groups in a regular expression.
  */
 var MATCHING_GROUP_REGEXP = /\((?!\?)/g;
+var MATCHING_NAMED_GROUP_REGEXP = /\(\?<(.+?)>/g;
```

Comment:
LGTM, should we just take the whole thing from the latest version? https://github.com/pillarjs/path-to-regexp/blob/c7ec332e87d8560673884d5629e1cb23cb03cb87/src/index.ts#L454-L468

Reply:
Good point. I combined both regex'es like in latest version.